cypress/integration/app.spec.js

```diff
@@ -0,0 +1,120 @@
+import { PRODUCT, CHARGE } from "../../src/ts/utils/constants";
+
+// describe("상품 관리 탭 테스트", () => {
+//   beforeEach(() => {
+//     cy.visit("http://localhost:9000/#product");
+//     cy.get(".product-control-input").eq(0).type("사이다");
+//     cy.get(".product-control-input").eq(1).type(2000);
+//     cy.get(".product-control-input").eq(2).type(20);
+//     cy.get("#product-add-button").click();
+//   })
+// 
+//   it("자판기에 상품 추가가 기능하다", () => {
+//     cy.get(".product-name").should("have.text", "사이다");
+//   });
```

Comment:
```suggestion
  const product = {
    name: "사이다",
    price: 2000,
    quantity: 20,
  };

  beforeEach(() => {
    cy.visit("http://localhost:9000/#product");
    cy.get(".product-control-input").eq(0).type(product.name);
    cy.get(".product-control-input").eq(1).type(product.price);
    cy.get(".product-control-input").eq(2).type(product.quantity);
    cy.get("#product-add-button").click();
  });

  it("자판기에 상품 추가가 기능하다", () => {
    cy.get(".product-name").should("have.text", product.name);
  });
```
테스트에 사용하는 데이터는 별도의 변수에 넣어두면 더 가독성이 좋다고 생각해요.
alias가 없어 2000, 20의 데이터가 어떤 역할을 하는지 더 파악하기 힘들어요